install/installer/pkg/components/public-api-server/configmap.go

```diff
@@ -135,7 +135,7 @@ func getPersonalAccessTokenSigningKey(cfg *experimental.Config) (corev1.Volume,
 	mount = corev1.VolumeMount{
 		Name:      "personal-access-token-signing-key",
 		MountPath: personalAccessTokenSigningKeyMountPath,
-		SubPath:   "signing-key",
+		SubPath:   "personal-access-token-signing-key",
```

Comment:
Curious why we need a subpath mount here at all. If the secret key is also good as a filename then we can just mount the secret directly without a subpath, right?

Reply:
Yeah that'd work as well. The difficulty is in sharing the right "secret field key" with the configmap, but that's also "fixable". Let me TAL

Reply:
Actually, I'll land this PR and investigate in a follow-up, to unblock staging.